.eslintrc.cjs

```diff
@@ -4,7 +4,4 @@ module.exports = {
   ignorePatterns: ["apps/**", "packages/**"],
   extends: ["@ucw-app/eslint-config/base"],
   parser: "@typescript-eslint/parser",
-  parserOptions: {
-    project: true,
-  },
```

Comment:
My linter suggested I remove this.

Reply:
I feel like the way I set up the linting in the project isn't super great. I'm not sure we're using the `packges/eslint-config` the way it's supposed to be used.

Reply:
I found this https://typescript-eslint.io/blog/parser-options-project-true/#introducing-true. I may try using `parserOptions.tsconfigRootDir` to tell it which directory to look in.